openebs-operator-1.11.0-aws.yaml

```diff
@@ -0,0 +1,752 @@
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
```

Comment:
@ranjithwingrider -- trying to setup a new convention for adding these file, so that we can automate them in future. Can you please check if following can be done: 
-  place this file under the folder /1.11.0/ and rename the file as openebs-operator-1.11.0-aws.yaml
-  Also place a copy of the same file in the main director as `openebs-operator-aws.yaml`

The above will allow, for checking the diff between last version and this version, and also the main directory will always list the latest aws operator yaml file.

Reply:
@kmova @muratkars I have done with the suggested changes. Please take a look at it. Thank you... 
I am not sure about how we can fully automating this procedure since it requires the new container group id for latest images.